### DISCUSSION TITLE
Firefox浏览器商店的插件版本回滚

### DISCUSSION THREAD

**Discussion by oswald522** [immersive-translate/immersive-translate#657] [Help]
### 概述 | Summary

Firefox浏览器商店的插件版本回滚至0.3.10
[插件地址](https://addons.mozilla.org/zh-CN/firefox/addon/immersive-translate/)

**Comment by theowenyoung** [comment]
Firefox 商店正在审查这个扩展，已按照他们的意见提交修改。
